holopy/core/io/io.py

```diff
@@ -258,8 +247,7 @@ def load_average(filepath, refimg=None, wavelen=None, index=None, polarization=N
     if len(filepath) < 1:
         raise LoadError(filepath, "No images found")
 
-    refimg = interpret_args(refimg, spacing, wavelen, index, polarization, optics)
-    accumulator = load_image(filepath[0], refimg.spacing, optics=refimg.optics)
+    accumulator = load_image(filepath[0], refimg.spacing, wavelen, index, polarization)
```

Comment:
Optical parameters as it is currently specified is not the right tool for this because it does conversions to wavevec. You may be right that a simplified version of interpret_args may need to come back, I hacked the old one out because it was mostly doing marshalling of optics objects, but some of the functionally might still be needed here.